cn/docs/tasks/inject-data-application/define-environment-variable-container.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 为容器设置环境变量
+---
+
+{% capture overview %}
+
+本页将展示如何为kubernetes Pod下的容器设置环境变量。
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+
+{% capture steps %}
+
+## 为容器设置一个环境变量
+
+创建Pod时，可以为其下的容器设置环境变量。通过配置文件的`env`或者`envFrom` 字段来设置环境变量。
+
+本示例中，将创建一个只包含单个容器的Pod。Pod的配置文件中设置环境变量的名称为`DEMO_GREETING`，
+其值为`"Hello from the environment"`。下面是Pod的配置文件内容:
+
+{% include code.html language="yaml" file="envars.yaml" ghlink="/docs/tasks/inject-data-application/envars.yaml" %}
```

Comment:
You might want to copy the **envars.yaml** file into the `/cn/docs/tasks/inject-data-application/` directory and update the path here. That should fix the Netlify preview.